lib/crystalball/rspec/runner.rb

```diff
@@ -17,13 +17,23 @@ def run(args, err = $stderr, out = $stdout)
 
           Crystalball.log :info, "Crystalball starts to glow..."
           prediction = build_prediction
-
+          dry_prediction(prediction)
```

Comment:
We should not proceed after printing the prediction if the flag is passed, so it would be something like
```ruby
if dry_run?
  puts prediction.to_a
  exit
end
```

Reply:
I made slightly change on it, hopefully that what you meant.